src/PROPOSAL/PROPOSAL/math/Spherical3D.h

```diff
@@ -34,4 +34,22 @@ namespace PROPOSAL  {
     protected:
         void print(std::ostream&) const override;
      };
+
+
+    /*!
+     * Container for a spherical vector with radius 1
+     *
+     * \param zenith
+     * \param azimuth
+     */
+     struct UnitSphericalVector{
+        double zenith;
+        double azimuth;
+    
+        constexpr UnitSphericalVector(const UnitSphericalVector&) = default;
+        constexpr UnitSphericalVector(double zenith, double azimuth)
+            : zenith(zenith), azimuth(azimuth) {};
+        constexpr UnitSphericalVector(const Spherical3D& s)
+            : zenith(s.GetZenith()), azimuth(s.GetAzimuth()) {};
```

Comment:
For this constexpr copy cstr, the getter functions must also be constexpr functions.

Reply:
well, it worked with Apple Clang 10